src/Data/Symbol.purs

```diff
@@ -5,20 +5,21 @@ module Data.Symbol
   , SProxy(..)
   ) where
 
+import Type.Proxy (Proxy(..))
+
 -- | A value-level proxy for a type-level symbol.
 data SProxy (sym :: Symbol) = SProxy
 
 -- | A class for known symbols
 class IsSymbol (sym :: Symbol) where
-  reflectSymbol :: SProxy sym -> String
+  reflectSymbol :: Proxy sym -> String
```

Comment:
This can potentially be:

```
reflectSymbol :: forall proxy. proxy sym -> String
```

And old code will continue to work.

Reply:
Thanks. I forgot about this solution.